README.md

```diff
@@ -302,6 +302,20 @@ At least, this is according to our understanding of the logic in the Go
 code. Ideally we should test this against `lyft/ratelimit` itself, which is
 a pending task.
 
+
+## Differences to Lyft's rate limit service
+
+Fencer differs in several ways compared to [Lyft's rate limit
+service](https://github.com/lyft/ratelimit/):
+
+* Fencer does not use Redis.
+* Fencer is implemented in Haskell, while `lyft/ratelimit` is
+  implemented in Go.
+* In Fencer, the `limitRemaining` key is left out altogether if the
```

Comment:
It's the other way round – lyft/ratelimit omits `limitRemaining` from the response if it's zero, and we always return it.

Reply:
Thanks for catching this, @neongreen! I fixed it.